Roles/AddOns/Common/Rainbow.cs

```diff
@@ -0,0 +1,52 @@
+namespace TOHE.Roles.AddOns.Common
+{
+    public static class Rainbow
+    {
+        private static readonly int Id = 27400;
+        public static bool IsEnable = false;
+        public static OptionItem CrewCanBeRainbow;
+        public static OptionItem ImpCanBeRainbow;
+        public static OptionItem NeutralCanBeRainbow;
+        //public static OptionItem RainbowColorChangeTime;
+        public static void SetupCustomOptions()
+        {
+            //RainbowColorChangeTime = FloatOptionItem.Create(27303, "RainbowColorChangeTime", new(0.1f, 5f, 2.5f), 0.5f, TabGroup.Addons, false)
+            //    .SetParent(CustomRoleSpawnChances[CustomRoles.Rainbow])
+            //    .SetValueFormat(OptionFormat.Seconds);
+            Options.SetupAdtRoleOptions(Id, CustomRoles.Rainbow, canSetNum: true, tab: TabGroup.Addons);
+            CrewCanBeRainbow = BooleanOptionItem.Create(Id + 10, "CrewCanBeRainbow", true, TabGroup.Addons, false)
+                .SetParent(Options.CustomRoleSpawnChances[CustomRoles.Rainbow]);
+            ImpCanBeRainbow = BooleanOptionItem.Create(Id + 11, "ImpCanBeRainbow", true, TabGroup.Addons, false)
+                .SetParent(Options.CustomRoleSpawnChances[CustomRoles.Rainbow]);
+            NeutralCanBeRainbow = BooleanOptionItem.Create(Id + 12, "NeutralCanBeRainbow", true, TabGroup.Addons, false)
+                .SetParent(Options.CustomRoleSpawnChances[CustomRoles.Rainbow]);
+        }
+        public static void OnFixedUpdate(PlayerControl player)
+        {
+            if (!IsEnable) return;
```

Comment:
This code is not needed since `IsEnabled` will already be checked
![Uploading image.png…]()